<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chpass</class>
 <widget class="QWidget" name="chpass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>418</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>451</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Arimo</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Смена пароля пользователя</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Пользователь:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lb_user">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Пароль:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="text_Password">
        <property name="font">
         <font>
          <family>Arimo</family>
          <pointsize>48</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Длина:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="text_number"/>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>символов</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Алфавит:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="check_b">
        <property name="text">
         <string>буквы</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="check_num">
        <property name="text">
         <string>цифры</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="check_sim">
        <property name="text">
         <string>символы</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="pb_NewPass">
        <property name="text">
         <string>Новый пароль</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_ChangePass">
        <property name="text">
         <string>Изменить пароль</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="pb_print">
        <property name="text">
         <string>Распечатать</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_AddToFile">
        <property name="text">
         <string>Добавить пароль в файл для печати</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
